pytest_django/plugin.py

```diff
@@ -599,7 +599,8 @@ def _dj_autoclear_mailbox() -> None:
 
     from django.core import mail
 
-    del mail.outbox[:]
+    if hasattr(mail, "outbox"):
```

Comment:
This change LGTM. This is an autouse fixture and should just do nothing if locmem backend is not used, not error.